libs/langchain-azure-dynamic-sessions/package.json

```diff
@@ -40,7 +40,7 @@
   "dependencies": {
     "@azure/identity": "^4.2.0",
     "@langchain/core": "~0.2",
-    "uuid": "^9.0.1"
+    "uuid": "^10.0.0"
```

Comment:
This is probably a good idea, but can we do the individual integration packages in a separate PR?

Reply:
Done. Added a bump of the package at https://github.com/langchain-ai/langchainjs/pull/5969.